examples/src/main/java/alluxio/cli/TestRunner.java

```diff
@@ -42,6 +43,10 @@
       .asList(WriteType.MUST_CACHE, WriteType.CACHE_THROUGH, WriteType.THROUGH,
           WriteType.ASYNC_THROUGH);
 
+  @Parameter(names = "--directory",
+      description = "The directory to test.")
```

Comment:
We're not testing the directory, we're using it as place to write files to for the purpose of testing Alluxio.